src/main/java/edu/wpi/first/wpilibj/networktables/NetworkTableValue.java

```diff
@@ -0,0 +1,325 @@
+package edu.wpi.first.wpilibj.networktables;
+
+import java.util.Objects;
+
+/**
+ * A network table entry value.
+ */
+public class NetworkTableValue {
```

Comment:
Just making sure that that's the sort of forwards compatibility guarantee you want to support in your API.